source/standards/dns-hosting.html.md.erb

```diff
@@ -8,24 +8,13 @@ review_in: 12 months
 
 To ensure users can connect to your service, you'll need accurate Domain Name System (DNS) records and a DNS provider to supply your DNS nameservers.
 
-You can find information on choosing where to host your DNS and how to request nameserver delegation at GDS in the [Service Manual][].
+For services [hosted on AWS](./hosting.html) you should use [Amazon Route 53][] as a cloud DNS web service.
 
-As part of your DNS strategy and planning you need to ensure:
+> [!INFO]
+> In the past some services have dual-hosted using [Google Cloud DNS][] for high nameserver availability.
```

Comment:
To be fair GOV.UK do still dual host DNS, but we've opted to not do it on certain zones that we operate that are not part of our core service.
I think (particularly as GOV.UK went offline due to using [a single DNS provider](https://en.wikipedia.org/wiki/DDoS_attacks_on_Dyn)) that we'd need to do more work to assess the risk before we did go back to single hosting.
We haven't had outages due to dual-hosting.

Reply:
I accept the point that this it is not necessary for all services to follow this approach.

Reply:
Ah, perhaps the reference in the GOV.UK DNS repo is a bit misleading then?

> Historically, zones were dual-hosted for redundancy, but this adds complexity to the point where it arguably causes more risk of an outage itself than the risk it is trying to mitigate.
> 
> Ideally, we would make things consistent by removing all zones from Google Cloud, but this is more risk and effort than it's worth.

Reply:
Yep, I think that was more opinion than outright policy 👍

Reply:
The default in that repo is still to dual host - you can see [where that's been overridden](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-dns-tf%20gcp_disabled&type=code)

Reply:
Cool, I've suggested a wording tweak below, but feel free to hack at it further.

Reply:
Hopefully this is addressed by 242c4e1